src/Button/RotationButton/RotationButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { faArrowsRotate } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { DragRotateAndZoom } from 'ol/interaction.js';
+
+import { useMap } from '../../Hook/useMap';
+import ToggleButton, {
+  ToggleButtonProps
+} from '../ToggleButton/ToggleButton';
+
+
+export type RotationButtonProps =  Partial<ToggleButtonProps>;
+
+export const RotationButton: React.FC<RotationButtonProps> = ({
+  tooltip = 'Shift + Drag to rotate and zoom the map around its center',
+  pressedIcon = <FontAwesomeIcon icon={faArrowsRotate} />,
+  tooltipProps,
+  ...passThroughProps}
+) => {
+  const map = useMap();
+
+  if (!map) {
+    return <></>;
+  }
+  let action = new DragRotateAndZoom();
```

Comment:
This will create a new interaction on every render. And I'm not sure if it gets properly cleaned up by the `removeInteraction` in line 32

maybe we can rebase to the rg-future branch. There we can make use of the new `useOlInteraction` hook. See https://github.com/terrestris/react-geo/blob/rg-future/src/Button/MeasureButton/MeasureButton.tsx#L183

that hook should take care of removing the interaction properly

Reply:
follow-up merge request